core/platform/models.py

```diff
@@ -106,6 +106,9 @@ def import_models(cls, model_names):
             elif name == NAMES.opportunity:
                 from core.storage.opportunity import gae_models as opportunity_models # pylint: disable=line-too-long
                 returned_models.append(opportunity_models)
+            elif name == NAMES.platform_parameter:
+                from core.storage.platform_parameter import gae_models as parameter_models # pylint: disable=line-too-long
```

Comment:
Should we really create a new folder for this? Couldn't be the `PlatformParameterModel` part of storage/config/?

Reply:
Done. Moved them to storage/config/.